src/op_handlers/unimplemented.rs

```diff
@@ -0,0 +1,7 @@
+use crate::eravm_error::{EraVmError, OpcodeError};
+use crate::{opcode::Opcode, state::VMState};
+
+pub fn unimplemented(_vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmError> {
+    eprintln!("Unimplemented instruction: {:?}!", opcode.variant);
```

Comment:
why is this `print` necessary?

Reply:
Just to make sure we can easier spot where the execution ended